src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java

```diff
@@ -13,21 +13,21 @@
     public final String jurisdiction;
     public final String zipFileName;
     public final Classification classification;
-    public final List<String> docUrls;
+    public final List<Document> documents;
 
```

Comment:
This list is not really final as in "immutable". It's just final as in "it cannot be replaced with another list".